### PR TITLE
Add API-X to vagrant install; Resolves https://github.com/Islandora-C…

### PR DESCRIPTION
**Pull request by ruebot**
…LAW/CLAW/issues/504

See: https://github.com/Islandora-CLAW/CLAW/issues/504

# How to test:
* clone repo, and checkout issue-504 branch
* `vagrant up`
* You should have no errors in the build, and see something along the lines of:
```
==> default: Installing API-X
==> default: Installing API-X
==> default: 337 [sshd-SshClient[7382f612]-nio2-thread-2] WARN org.apache.sshd.client.keyverifier.AcceptAllServerKeyVerifier - Server at /0.0.0.0:8101 presented unverified RSA key: 6a:cc:33:3d:76:08:39:0a:34:1c:00:20:c2:d1:5a:1d
==> default: Adding feature url mvn:org.fcrepo.apix/fcrepo-api-x-karaf/0.1.0/xml/features
==> default: 
==> default: 
==> default: Running provisioner: shell...
    default: Running: /tmp/vagrant-shell20170126-11798-3ueq4q.sh
```
@dannylamb @whikloj can you test?

### CODE REVIEW

**Comment by dannylamb** [comment]
@ruebot I think we need to update fcrepo-camel.  Which will require updating camel to 2.18.x.  This will also force us to upgrade the amherst services we have running as well.

That's an oversight on my part.

**Comment by dannylamb** [comment]
@ruebot Are you ok with handling that explosion of scope?  We can address all those individually if you don't have the time right now.

**Comment by dannylamb** [comment]
@ruebot https://github.com/Islandora-CLAW/claw_vagrant/blob/master/configs/karaf/fcrepo_camel_toolbox.script#L1

It's a transitive dependency of fcrepo-camel-toolbox.  Try bumping the version to 4.7.0.

You can update Amherst stuff here: https://github.com/Islandora-CLAW/claw_vagrant/blob/master/configs/karaf/islandora_id_mapping_service.script#L1

**Comment by dannylamb** [comment]
@ruebot That sounds like a good course of action.  Carry on.  Just be sure to link both pulls in the issue.

**Comment by dannylamb** [comment]
@ruebot changes are looking good.  I'll spin up a copy as soon as I can and test.

File: configs/karaf/fcrepo_camel_toolbox.script [hunk 1]
```diff
@@ -1,3 +1,5 @@
-feature:repo-add mvn:org.fcrepo.camel/toolbox-features/4.6.2/xml/features
+feature:repo-add camel 2.18.1
+feature:repo-add activemq 5.14.0
+feature:repo-add mvn:org.fcrepo.camel/toolbox-features/4.7.0/xml/features
```
Comment (dannylamb) [inline configs/karaf/fcrepo_camel_toolbox.script:3]: If we install Alpaca first, we don't need these feature declarations.  Alpaca's features.xml will take care of it for us and you can delete these three lines.

File: configs/karaf/islandora_id_mapping_service.script [hunk 2]
```diff
@@ -1,4 +1,4 @@
-feature:repo-add mvn:edu.amherst.acdc/acrepo-karaf/1.0.6/xml/features
+feature:repo-add mvn:edu.amherst.acdc/acrepo-karaf/1.0.8/xml/features
 feature:install acrepo-services-activemq
```
Comment (dannylamb) [inline configs/karaf/islandora_id_mapping_service.script]: This has been replaced with fcrepo-service-activemq.  If this script gets run after `alpaca.sh`, we'll be fine to delete this.

**Review by dannylamb — CHANGES_REQUESTED** [review]
Something came up when working on https://github.com/Islandora-CLAW/Alpaca/pull/28.  The vagrant environment needs to be able to resolve Drupal urls, which we can't do at the moment because of our port forwarding strategy.  Can you please run apache on port 8000 so that localhost:8000 resolves on both guest and host machine?

**Comment by dannylamb** [comment]
`vagrant up`

File: configs/variables [hunk 3]
```diff
@@ -18,4 +18,4 @@ FITS_WS_VERSION=1.1.3
 MYSQL_USER=root
 MYSQL_PASS=islandora
 MYSQL_CONNECTOR_VERSION=6.0.4
-GITHUB_TOKEN=changeme
+GITHUB_TOKEN=f71459f666cabb3588d019b4d11e54e0befbda72
```
Comment (DiegoPino) [inline configs/variables]: @ruebot are you giving your token away here?

**Review by dannylamb — APPROVED** [review]

**Pull request merged**